pipeline_dp/dataset_histograms/computing_histograms.py

```diff
@@ -117,6 +129,46 @@ def bins_to_histogram(bins):
     return backend.map(col, bins_to_histogram, "To histogram")
 
 
+def _compute_frequency_histogram_helper_with_lowers(
```

Comment:
I will about better design to avoid this code duplication, for now it is just a draft